typescript/src/folder/ui-folder-info/FolderInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./FolderInfo.module.scss";
+import classNames from "classnames/bind";
+
+const cx = classNames.bind(styles);
+
+interface props {
+  profileImage: string | undefined;
```

Comment:
특정 타입 + undefined라는 타입을 표현할때는 이렇게 해주면 됩니다~
```suggestion
  profileImage?: string;
```